src/Microsoft.AspNetCore.Mvc.Core/ApplicationParts/AssemblyPart.cs

```diff
@@ -42,6 +51,27 @@ public AssemblyPart(Assembly assembly)
         public IEnumerable<TypeInfo> Types => Assembly.DefinedTypes;
 
         /// <inheritdoc />
+        public IReadOnlyCollection<PrecompiledViewInfo> PrecompiledViews
```

Comment:
Should we do `IEnumerable` or `IReadOnlyList` here? We don't really use `IReadOnlyCollection` anywhere else, and there isn't a reason to prefer it over `IEnumerable`